### PR TITLE
Image with TLS enabled for Kubernetes application.

### PR DESCRIPTION
**Pull request by tomasz-safuryn**

### CODE REVIEW

File: 9/9.6/kubernetes-app/tls-configuration.sh [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+		tlslocation=$pgdata/$POSTGRES_TLS_LOCATION
+	else
+		tlslocation=$POSTGRES_TLS_LOCATION
+	fi
+fi
+
+if [[ -d $tlslocation ]]; then
+	if [[ ! -f $tlslocation/server.crt || ! -f $tlslocation/server.key ]]; then
+		echo "There are no $tlslocation/server.crt or $tlslocation/server.key files."
+		exit
```
Comment (jprzychodzen) [inline 9/9.6/kubernetes-app/tls-configuration.sh:42]: Is this an error and we should exit with '1'?

File: 9/9.6/kubernetes-app/tls-configuration.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	fi
+	echo "Using $tlslocation/server.crt and $tlslocation/server.key files."
+	cp $tlslocation/server.crt $tlslocation/server.key $pgdata
+	chmod 600 $pgdata/server.crt $pgdata/server.key
```
Comment (jprzychodzen) [inline 9/9.6/kubernetes-app/tls-configuration.sh:46]: Do we need to set an ownership with chown?

**Review by jprzychodzen — APPROVED** [review]

File: 9/9.6/kubernetes-app/tls-configuration.sh (hunk 2 above)
Comment (tomasz-safuryn) [reply 9/9.6/kubernetes-app/tls-configuration.sh:46]: No

File: 9/9.6/kubernetes-app/tls-configuration.sh (hunk 1 above)
Comment (tomasz-safuryn) [reply 9/9.6/kubernetes-app/tls-configuration.sh:42]: It's not an error.

**Pull request merged**